lms/services/annotation_activity_email.py

```diff
@@ -0,0 +1,49 @@
+from dataclasses import asdict
+
+from sqlalchemy import select
+
+from lms.models import Assignment, LMSUser
+from lms.services.mailchimp import EmailRecipient, EmailSender
+from lms.tasks.mailchimp import send
+
+
+class AnnotationActivityEmailService:
+    """Service to send emails for annotation activity (eg. mentions)."""
+
+    def __init__(self, db, sender):
+        self._db = db
+        self._sender = sender
+
+    def send_mention(self, mentioned_user_h_userid, assignment_id: int):
```

Comment:
Not much going on here, just get the relevant objects from the DB and send the email.